{% set title = 'Help' %} 
{% extends 'layout.html' %} 

{% block head %}
{{super()}}
<link rel="stylesheet" href="{{ url_for('static', filename='dist/css/help.css') }}">
{% endblock %}


{% block content %}
<div class="container">
	<div class="page-header">
		<h1>FuncTree2 Documentation</h1>
	</div>
	<section id="introduction" class="page-header">
		<div class="row">
			<div class="col-sm-6">
				<p class="lead justify">
					FuncTree2 allows you to visualize and explore omics data in the
					hierarchical context of biological ontologies. Mapping omics
					data onto a pre-defined interactive treemap, based on the
					<a href="http://www.genome.jp/kegg-bin/get_htext?br08902.keg"
						target="_blank">KEGG brite functional hierarchy</a>, lets you
					quickly and comprehensively understand the functional potential of
					your data, which will help you develop further hypothesis and
					scientific insights. Statistical tests, module pathway
					reconstruction, and different customization controls are built-in
					to help you in your quest.
				</p>
			</div>
			<div class="col-sm-6">
				<img alt=""
					src="{{ url_for('static', filename='images/functree-kegg-overview.png') }}"
					width="300px">
			</div>
		</div>
		<hr />
		<div class="row">
			<div class="col-sm-offset-3 col-sm-6">
				<p style="text-align: center; font-size: 16px; font-style: italic; color: grey;">
					Here you will learn about the different types of analysis available
					in FuncTree2 and the accepted <a href="#data-format">data formats</a>.
					We also give you some usage tips and a sneak preview of what your 
					<a href="#example">output</a> will look like.
				</p>
			</div>
		</div>
	</section>
	<section id="data-format" class="page-header">
		<h2>Data format</h2>
		<h3>Tab separated matrices</h3>
		<div class="row">
			<div class="col-sm-6">
				<table id="omics-matrix" class="table">
					<caption style="caption-side: bottom; text-align: left;">
						Snippet of input data matrix.
						 <span>
						 	<a href="{{ url_for('static', filename='data/example/profile.tsv') }}"
							target="_blank"><i class="fa fa-download" aria-hidden="true"></i>
								Full example</a></span>
					</caption>
					<thead>
						<tr role="row">
							<th>Annotation</th>
							<th>Sample1</th>
							<th>Sample2</th>
							<th>Sample3</th>
							<th>..</th>
							<th>sampleN</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<td>K00001</td>
							<td>1</td>
							<td>2</td>
							<td>5</td>
							<td>..</td>
							<td>10</td>
						</tr>
						<tr>
							<td>K00002</td>
							<td>1</td>
							<td>2</td>
							<td>9</td>
							<td>..</td>
							<td>10</td>
						</tr>
						<tr>
							<td>K00302</td>
							<td>31</td>
							<td>7</td>
							<td>12</td>
							<td>..</td>
							<td>0</td>
						</tr>
						<tr>
							<td>..</td>
							<td>..</td>
							<td>..</td>
							<td>..</td>
							<td>..</td>
							<td>..</td>
						</tr>
						<tr>
							<td>K00107</td>
							<td>21</td>
							<td>14</td>
							<td>11</td>
							<td>..</td>
							<td>170</td>
						</tr>
					</tbody>
				</table>
			</div>
			<div class="col-sm-6">
				<p class="justify">A Tab separated Omics matrix, annotated with
					one of the following ontology databases is accepted:</p>
				<ul class="list-group">
					<li class="list-group-item list-group-item-success">KEGG KO</li>
					<li class="list-group-item list-group-item-success">KEGG
						Modules</li>
					<li class="list-group-item list-group-item-info">NCBI gene ids</li>
				</ul>
				<p>In addition an optional Tab separated color list could be uploaded to color associated annotations in circle mode .</p>
				<span><a data-toggle="collapse" href="#color-list"><i
						class="fa fa-eye" aria-hidden="true"></i> Display color list
						snippet</a></span>&nbsp; <span><a
					href="{{ url_for('static', filename='data/example/color.tsv') }}"
					target="_blank"><i class="fa fa-download" aria-hidden="true"></i>
						Full example</a></span>
				<table id="color-list" class="table table-hover table-striped dataTable no-footer collapse">
					<thead>
						<tr role="row">
							<th>Annotation</th>
							<th>Color</th>
						</tr>
					</thead>
					<tbody>
						<tr>
							<td>Translation</td>
							<td>#ff0000</td>
						</tr>
						<tr>
							<td>map03010</td>
							<td>#ff0000</td>
						</tr>
						<tr>
							<td>..</td>
							<td>..</td>
						</tr>
						<tr>
							<td>Sample2</td>
							<td>#ffffff</td>
						</tr>
					</tbody>
				</table>
			</div>
		</div>
		<h3>JSON</h3>
		<div class="row">
			<div class="col-sm-12">
				<br />
				<p class="justify">Processed datasets could be exported in JSON 
				<span><a href="{{ url_for('static', filename='data/example/profile.json') }}"
						target="_blank"> <i class="fa fa-download" aria-hidden="true"></i> format
					</a></span> and <a href="{{ url_for('route_analysis', mode='display') }}">re-uploaded</a> later to FuncTree2.
					Below is an example of a valid JSON file and its schema definition.   
				</p>
			</div>
			<div class="col-sm-12">
				<div>
					<!-- Nav tabs -->
					<ul
						class="nav nav-tabs"
						role="tablist">
						<li
							role="presentation"
							class="active"><a
							href="#json-data-snippet"
							aria-controls="json-data-snippet"
							role="tab"
							data-toggle="tab">Example</a></li>
						<li role="presentation"><a
							href="#json-data-schema"
							aria-controls="json-data-schema"
							role="tab"
							data-toggle="tab">Schema</a></li>
					</ul>
					<!-- Tab panes -->
					<div class="tab-content">
						<div
							role="tabpanel"
							class="tab-pane active"
							id="json-data-snippet">
<pre class="pre-scrollable"><code data-language="json">{{ json_example() }}</code></pre>
						</div>
						<div
							role="tabpanel"
							class="tab-pane"
							id="json-data-schema">
<pre class="pre-scrollable"><code data-language="json">{{ json_schema() }}</code>
							</pre>
						</div>
					</div>
				</div>
				<span>
					<a href="{{ url_for('static', filename='data/example/data-example.json') }}" target="_blank">
					<i class="fa fa-download" aria-hidden="true"></i> Example</a>
				</span>&nbsp;
				<span>
					<a href="{{ url_for('static', filename='data/example/data-schema.json') }}" target="_blank">
					<i class="fa fa-download" aria-hidden="true"></i> Schema</a>
				</span>
			</div>
		</div>
	</section>
	<section id="workflow" class="page-header">

		<h2>Workflow</h2>

		<div class="row">
			<div class="col-sm-12">
				<img alt=""
					src="{{ url_for('static', filename='images/workflow.png') }}"
					width="960px">
			</div>
		</div>
		<ul>
			<li>Mapping: [Rename: to Ontology preocessing and mapping] =>
				maps KOs onto modules, pathways and brite hierarchies...
			<li>Module coverage: Module based mapping, where each module is
				given a coverage and abundance and then used for the pathway and
				brite abundance
			<li>Comparison: Two KO matrices, are used separatly for basinc
				mapping and the output is compared using Mann-Whitney U test
			<li>Display: Mapping without any processing. Lets the user take
				compltete control of generating his abundance data (Examnple)
			<li>0 size is log to penalize high p-values
			<li>FDR?
		</ul>
	</section>
	<section id="Viewer" class="page-header">
		<h2>Viewer</h2>
		<section id="functree">
			<h3>FuncTree</h3>
			<div class="row">
				<div class="col-sm-6">
					<p class="lead justify">
						FuncTree explorer turns an ontology database into an interactive
						<a href="https://github.com/d3/d3-hierarchy#tree">Radial Tidy Tree</a> where each layer of nodes is mapped
						to a level of the ontology. The outermost layer is always the lowest level visualized, while the innermost layer 
						is the highest level, and nodes represent biological entries (e.g. KO, Modules, pathways, ...).
						The four highest levels are shown by default, but more or less levels could be shown by modifying the depth of the tree
						in the option pane. Clicking on a node will collapse or reveals its children nodes, while right-clicking opens a context menu
						that offers rerooting of the tree by visualizing the selection as a Radial Tidy Tree in a new Viewer window. The context menu
						can display more details about the selection or copy it to clipboard. Mouse gestures enable zooming 
						(<kbd>middle&nbsp;button</kbd>) and panning (<kbd><kbd>click</kbd> + <kbd>drag</kbd></kbd>). A search with auto-complete is available
						to help quickly locating a biological entry of interest on the tree.
					</p>
					<span>Summary of features:</span>
					<ol>
						<li>Interactive Radial Tidy Tree
						<li>Tree re-rooting.
						<li>Zooming and panning
						<li>Search & highlight of biological entries
					</ol>
				</div>
				<div class="col-sm-6">
					<p>
						FuncTree viewer is highly customizable to reflect user data. For
						example KEGG KO abundance from one or multiple samples is
						visualized on the KO layer in the form of stacked bars or colored
						circles. Values for higher layers are also derived and visualized in
						the same fashion.
					</p>
					<h4>Data series selection</h4>
					<p>Data series are profiles that FuncTree can customize and
						project onto the reference tree. Series generated by FuncTree will
						always include an Average and a Sum based series. After series
						selection, you can chose which samples to highlight as stacked
						bars (optionally normalized aka 100% stacking) on the tree.
						Additionally, a sample of choice could also be mapped to the
						radius of each node.</p>
					<h4>Coloring options for circle mapping of one sample</h4>
					<ul>
						<li>Layer: Automatically genrated colors for Ontology layers
							e.g. Brite L1, L2, Pathway, Module
						<li>Column: use the color code associated with a sample in
							the input data
						<li>Entry: Reflect user predefined colors for each entry.
							Defaults to grey.
					</ul>
					<h4>Data series generation</h4>
					<p>
						FuncTree summarizes KO abundances on higher layers by taking the
						mean or the sum of the KOs mapped to an entity on the layer.
					</p> 
					<div>
						<ul>
							<li>Each node abundance is calculated as the mean of its mapped KOs normalized by its KO size.
							<li>In case sum only a sum is applied 
							<li>In case of modules the module coverage is visualized, then on parent levels the mean of module coverage is used (without normalization) 
						</ul>
					</div>
					<img
							alt=""
							class="img-responsive"
							style="width: 100%;"
							src="{{ url_for('static', filename='images/mapping-stacked-100.png') }}">
					</div>
			</div>
			<div class="row" style="padding-top: 20px;">
				<div class="col-sm-6">
					<img alt="" class="img-thumbnail img-fluid"
						src="{{ url_for('static', filename='images/mapping-stacked-100-zoom.png') }}">
				</div>
				<div class="col-sm-6">
					<img alt="" class="img-thumbnail img-fluid"
						src="{{ url_for('static', filename='images/explorer-rerooted.png') }}">
				</div>
			</div>
			<h4>Tree export</h4>
			<p>FuncTrees could be exported into SVG, PDF, PostScript, and PNG</p>
		</section>
		<section>
			<h3>Summary charts</h3>
			<div class="row">
				<div class="col-md-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/chart-sample-barplot-100.png') }}">
				</div>
				<div class="col-md-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/chart-sample-barplot.png') }}">
				</div>

				<div class="col-md-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/chart-pie.png') }}">
				</div>
				<div class="col-md-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/chart-heatmap.png') }}">
				</div>
			</div>
			
			<h4>Charts export</h4>
			<p>Charts could be exported into SVG, PDF, PNG, and JPEG</p>
		</section>
		<section>
			<h3>iPath integration</h3>
			<div class="row">
				<div class="col-sm-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/chart-sample-barplot-100.png') }}">
				</div>
				<div class="col-sm-6">
					<p>Input data could be also visualized in iPath</p>
					
					<p>TODO add different coloring options as well as level selection (KO, Module, Map)</p>
				</div>
			</div>
		</section>
		
		<section>
			<div class="row" style="display: flex;">
				<div class="col-sm-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/table.png') }}">
				</div>
				<div class="col-sm-6 col-sm-offset-1" style="display: flex;  align-items: center;">
					<p class="lead"><b>Summary Table</b><br /> Data series could be viewed in table format. FuncTree nodes are presented in the Entry column, given a layer value and a numerical 
					value for all samples in the series. Tables could be downloaded in TSV format for further exploration and manipulation offline.</p>
				</div>
			</div>
		</section>
		
		<hr>
		
		<section>
			<div class="row">
				<div class="col-sm-5">
					<h3>Profile download and sharing</h3>
					<p class="lead">Profiles could be dowloaded in JSON format for storage or sharing. The <a href="{{ url_for('route_analysis', mode='display') }}">display</a>
					function allows re-upload and display of the JSON profiles in FuncTree. Alternatively, profiles could be shared by a link. Clicking the share icon
					copies the link to clipboard and displays it on screen.</p>
				</div>
				<div class="col-sm-offset-1 col-sm-6">
					<img class="img-thumbnail img-fluid" alt=""
						src="{{ url_for('static', filename='images/summary.png') }}">
				</div>
			</div>
		</section>
	</section>
	<section class="page-header">
		<h2 id="example">Examples</h2>
		Add metagenomics, multiomics and pangenomics examples
	</section>
</div>
{% endblock %}
